Sources/Growth/URP/AttributionManager.swift

```diff
@@ -0,0 +1,203 @@
+// Copyright 2023 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import Foundation
+import Preferences
+import Combine
+import Shared
+
+public enum FeatureLinkageType: CaseIterable {
+  case notdefined, vpn, playlist, leoAI
+  
+  var adKeywords: [String] {
+    switch self {
+    case .vpn:
+      return ["vpn, 1.1.1.1"]
```

Comment:
what is 1.1.1.1 for?

Reply:
It is an actual keyword from vpn campaign :)